imports/plugins/included/actionableAnalytics/client/containers/Report.js

```diff
@@ -0,0 +1,297 @@
+import React from "react";
+import $ from "jquery";
+import _ from "lodash";
+import { Template } from "meteor/templating";
+import { Orders } from "/lib/collections";
+import { formatPriceString } from "/client/api";
+import { ReactiveDict } from "meteor/reactive-dict";
+import { registerComponent } from "@reactioncommerce/reaction-components";
+import { ReportSideBar } from "../components/ReportSideBar";
+import { RetailDashBoard } from "../components/RetailDashBoard";
+import { SalesReport } from "../components/SalesReport";
+import { InventoryReport } from "../components/InventoryReport";
+import { ProductStat } from "../components/ProductStat";
+import { analyseOrder } from "../helpers/analyseOrder";
+import { getAverageSales } from "../helpers/getAverageSales";
+import { Meteor } from "meteor/meteor";
+import { DateTimePicker } from "../components/datePicker/DateTimePicker";
+
+
+/**
+ * @export
+ * @class Report
+ * @extends {React.Component}
+ */
+export class Report extends React.Component {
+  constructor(props) {
```

Comment:
The constructor is not needed here as there's no initial state value that uses `props`. Therefore, delete the constructor and initialize state as:
```js
state = {
  value: "allProduct",
  statValue: "quantitySold",
  ordersPlaced: 0,
  ...
};